cypress/integration/comparator.spec.ts

```diff
@@ -1,2 +1,49 @@
 // TODO: implement e2e for comparator here
+
+// Write new test that fills out the form so that it ends up with the standard comparator query, to confirm our form is functional.
+
+// Ask Belco how to get cy.findByText(/Changes from v6.16.0 to v8.1.0/i).should('exist') to wait for the result.
+// We should mock/fixture the result somehow, as we're expecting the same results each time... and it's really easy to hit the API limits unauthenticated.
+// For now these tests do work properly, but they're not using any Testing Library queries.
+
+it('should show expected results when using standard query string', () => {
+  cy.visit(
+    '/?repo=testing-library%2Fdom-testing-library&from=v6.16.0&to=v8.1.0'
+  )
+
+  // Confirm from and to version range are displayed
+
+  cy.contains('Changes from v6.16.0 to v8.1.0')
```

Comment:
I tried replacing this with:

```suggestion
  cy.findByText(/Changes from v6.16.0 to v8.1.0/i).should('exist')
```

But it doesn't work because the `cy.findByText` function does not wait until we get the response from the real API... 😄 
Not sure how I tell Cypress/Testing Library to wait until we get a response before checking any of the conditions in the test.

Reply:
When we have stubbed the requests, then we can wait for them to be completed like this: https://docs.cypress.io/guides/guides/network-requests#Waiting

Reply:
Might as well give me an example of telling Cypress to wait as well, if you're going to help with this as well! 😄